extensions/quick-open-project/package.json

```diff
@@ -1,91 +1,105 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
-  "name": "quick-open-project",
-  "version": "1.1.0",
-  "title": "Quick Open Project",
-  "description": "Find and open projects on your system",
+  "name": "remove-paywall",
```

Comment:
I am guessing this is a copy/paste error and you don't actually wish to change the plugin name, description, etc?

Reply:
Correct 🫠